src/main/java/frc/robot/commands/elevator/ManualElevatorControl.java

```diff
@@ -0,0 +1,57 @@
+//TODO: Need to figure out a way to ovverride the constant PID adjustment happening in Elevator.java subsystem to run this command
+
+
+package frc.robot.commands.elevator;
+
+import frc.robot.controls.Operator;
+import frc.robot.controls.TestController;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Elevator;
+
+public class ManualElevatorControl extends CommandBase {
+  Elevator m_elevator; 
+  double m_currentElevatorPos;
+  TestController m_testController;
+  /**
+   * This command allows for manual control of the elevator. 
+   * TODO: get manual control of all subsystems to be exclusive to a third controller
+   * @param elevator
+   */
```

Comment:
this TODO probobly shouldn't be in This Command and definitely shouldn't be the java doc.